index.html

```diff
@@ -0,0 +1,210 @@
+            <span class="large-screen">In 2016 Debru and Dilsizian founded Nyege Nyege<br>
+            Tapes to release regional, non-commercial African music for a wider audience</span></p>
+          </div>
+        </div>
+      </footer>
+    </div>
+    <!--  about page begins here-->
+    <div class="about-us hidden">
+      <section class="about-us-title">
+        <p class="tagline tagline-2 small-screen">"MTN Nyegenyege welcomes you all"</p>
+        <h2 class="page-title page-title-2 pad">THE MTN NYEGENYEGE FESTIVAL 2022</h2>
+        <p class="introduction highlight pad">
+          Artists affiliated with Nyege Nyege have toured throughout 
+          Europe and Asia, playing festivals including CTM and Unsound. 
+          The 2019 edition of Red Bull Music Festival in New York
+          featured a night dedicated to the collective, although the 
+        </p>
+        <div class="contact-info">
+          <p>Please contact us through Email for any questions about 
+            the Nyegenyege festival 2022
+          </p>
+          <p><b class="bold-text">Nyegenyege2022@mtn.ug.org</b></p>
+        </div>
+      </section>
+      <section class="topic-logo">
+          <h3 class='logo-h3'>MTN Nyegenyge 2022 festival logo</h3>
+          <span class="guide-bar red-line"></span>
+          <p class='logo-p'>The logo of MTN Nyegenyege 2022 festival was decided through 
+            the logo competion from February 12th to 5th March</p>
+          <div class="the-logo">
+            <a href="#" class="brand"><img src="images/logo.png" alt="logo"  class="brand">Nyegenyege</a>
+          </div>
+      </section>
+      <section class="past-festivals">
+        <h3 class="logo-h3 festivals-h3">See the past Nyegenyege festivals</h3>
+        <span class="guide-bar"></span>
+        <p >Take a look at the last two festivals that took place in Jinja at the nile and Kampala</p>
+        <div class="key-moments">
+          <div class="past-event">
+            <img src="images/nyege1.jpg" alt="people at the nyegenyege festival in 2021">
+            <div class="filter-layer">
+              <p>2021<br/>Nyegenyege Festival in Jinja</p>
+            </div>
+          </div>
+          <div class="past-event year2020">
+            <img src="images/nyege2.jpg" alt="a man lifting a lady at the nyegenyege festival in 2020">
+            <div class="filter-layer">
+              <p>2020<br/>Nyegenyege Festival in Kampala</p>
+            </div>
+          </div>
+        </div>
+      </section>
+      <footer class="page-footer">
+        <section class="partner-logos">
+          <h3>Partner</h3>
+          <span class="guide-bar"></span>
+          <div class="partners">
+            <p class="mtn">MTN</p>
+            <p class="airtel">Airtel</p>
+            <p class="capital">Capital FM</p>
+            <p class="sanyu">Sanyu Radio</p>
+            <p class="silk">Silk Events</p>
+          </div>
```

Comment:
The error at your `feature speaker section` prevents your `footer` from loading. So, I am unable to review the `footer`, `speaker section`, and `about page`. Kindly fix the error. Also, I am unable to navigate to your `about page`.